tests/Fixer/Whitespace/NoSpacesBetweenOffsetFixerTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Whitespace;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ *
+ * @internal
+ */
+final class NoSpacesBetweenOffsetFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFixSpaceInsideOffset($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function testLeaveNewLinesAlone()
```

Comment:
I've updated this method name. I forgot to rename it after I copied the example from another fixer.